src/libraries/System.IO.FileSystem/tests/Path/Exists_Directory.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class PathDirectory_Exists : Directory_Exists
+    {
+        public override bool Exists(string path) => Path.Exists(path);
+
+        #region Universal
```

Comment:
I know that the existing tests use regions, but that is mostly because they are very old.  Since regions don't add any value in the new types that you have created here (two methods in total), please remove them (only in the new files to keep the diff small).

Reply:
Fixed